package.json

```diff
@@ -7,6 +7,9 @@
     "type": "git",
     "url": "https://github.com/omphalos/bayesian-bandit.js.git"
   },
+  "scripts": {
+    "test": "NODE_ENV=test nodeunit tests.js"
```

Comment:
For most of my projects I run tests under the "test" environment.  It doesn't really add anything here though so I'll go ahead and remove.  It mostly matters when there's differences b/w prod / test / local environments and we don't have anything like that here.